hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneIllegalArgumentException.java

```diff
@@ -37,4 +37,8 @@
   public OzoneIllegalArgumentException(final String message) {
```

Comment:
General question why are specializing IllegalArgumentException if not other state/behavior added to it :)

Reply:
Good point. Removed this from current patch.